articles/machines/backend-simulators.md

```diff
@@ -26,6 +26,22 @@ IonQ provides a GPU-accelerated idealized simulator supporting up to 29 qubits,
 
 For more information, see the [IonQ provider](xref:microsoft.quantum.providers.ionq) page.
 
+## QCI
+
+The QCI simulator is designed to model QCI hardware. Although the simulator is
+ideal with regard to quantum noise, it accurately reflects precision and
+accuracy of classical calculations on 18 bit signed arithmetic on integers and
+fixed-point registers. Before simulating, it completes a full compilation and
+thereby validates the program against hardware constraints.
+
+<!-- ADD ANY CONTENT THAT YOU WANT TO USE TO DESCRIBE YOUR SERVICES AND COMPANY -->
+
+- Job type: `Simulation`
+- Data Format: `microsoft.qir.v1`
+- Target ID: `qci.simulator`
+- Target Execution Profile: Basic Measurement Feedback
```

Comment:
I'm not sure if this profile is right, e.g., fixed point registers, but I'm not sure what would be more accurate.

Reply:
@ffrachon @l-enloe @qci-petrenko

Reply:
@cesarzc

Reply:
The target execution profile should be Apdative Execution:
`Target Execution Profile: Adaptive Execution`

Reply:
@cesarzc do you have different guidance on what we should say for profile given our conclusions yesterday?

Reply:
this can be closed